lib/ui/onboarding/import_account/usecases/find_account_use_case.dart

```diff
@@ -16,7 +17,7 @@ class FindAccountsUseCase extends InputUseCase<Result<Iterable<UserProfileData>,
   @override
   Future<Result<Iterable<UserProfileData>, HyphaError>> run(String input) async {
     final eosClient = EOSClient(
-      baseUrl: remoteConfigService.pushTransactionNodeUrl(),
+      baseUrl: remoteConfigService.baseUrl(),
```

Comment:
when do we use pushTransactionNodeUrl and when baseUrl

Reply:
Always baseURL, except when we want to push a transaction.

So if we use this URL to sign a transaction and send it off, we use pushTransactionNodeURL

The idea is that that URL points to a fast node - like greymass

A lot of nodes don't have all the features though so for all read operations, and search accounts also, we use baseURL which is set to the load balancer, which automatically picks a node that has what we need for the call in question. 

All nodes have normal chain lookups - like loading table data. 

Only some nodes have a history plugin

And only some nodes have the account lookup - not sure if that is part of history plugin. I didn't even know normal nodes don't have this by default until it failed ...